msg/actuator_controls.msg

```diff
@@ -9,8 +9,17 @@ uint8 INDEX_FLAPS = 4
 uint8 INDEX_SPOILERS = 5
 uint8 INDEX_AIRBRAKES = 6
 uint8 INDEX_LANDING_GEAR = 7
+uint8 INDEX_GIMBAL_SHUTTER = 3
+uint8 INDEX_CAMERA_ZOOM = 4
+
 uint8 GROUP_INDEX_ATTITUDE = 0
 uint8 GROUP_INDEX_ATTITUDE_ALTERNATE = 1
+uint8 GROUP_INDEX_GIMBAL = 2
+uint8 GROUP_INDEX_MANUAL_PASSTHROUGH = 3
+uint8 GROUP_INDEX_MC_VIRTUAL = 4
+uint8 GROUP_INDEX_FW_VIRTUAL = 5
```

Comment:
@sfuhrer are these correct?

Reply:
Looking at the guide (https://dev.px4.io/master/en/concept/mixing.html) or [here](https://github.com/PX4/Firmware/blob/abd2bb4eb738eef2007c43206a353ca38d4fdf6e/msg/actuator_controls.msg#L18): yes, ordering is correct. But not sure where we would need the GROUP_INDEX of MC_VIRTUAL and FW_VIRTUAL atm.

Reply:
@julianoes @dayjaby What needs to happen to progress this? I'm just working through my docs PRs and this is a dependency.